packages/web-service/src/handlers/__tests__/submit-application.spec.js

```diff
@@ -0,0 +1,33 @@
+describe('the submit-application handler function', () => {
+  beforeEach(() => jest.resetModules())
+
+  it('correctly calls the API and submits the journey data', async () => {
+    const mockGetData = jest.fn(() => ({
+      userId: 'afda812d-c4df-4182-9978-19e6641c4a6e',
+      applicationId: '35a6c59e-0faf-438b-b4d5-6967d8d075cb'
+    }))
+    const request = {
+      cache: () => ({
+        getData: mockGetData
+      })
+    }
+    const mockSubmit = jest.fn()
+    jest.doMock('../../services/api-requests.js', () => ({
+      APIRequests: {
+        APPLICATION: {
+          submit: mockSubmit
+        }
+      }
+    }))
+    const mockRedirect = jest.fn(() => 'next')
+    const h = {
```

Comment:
Is this the response mock? `h` is a vague variable name

Reply:
'h' is the generic name given to the hapi response toolkit. It is called 'h' in the documentation. Yes, it a poor name! responseToolkit would have been better, but I think it should align with the documentation. I am keen on canonical usage wherever possible.